frontend/src/utilities/sortByCreatedAt.ts

```diff
@@ -0,0 +1,2 @@
+export const sortByCreatedAt = (pipeline) =>
+  pipeline.sort((a, b) => new Date(b.created_at).getTime() - new Date(a.created_at).getTime());
```

Comment:
If this method is targeted only for pipelines, add it in the `utils` file within pipelines, if not, make this method generic.